### PR TITLE
T25 - Sort and swap special character 

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+Write ("\nEnter the order of sorting (D)escending or (O) for Optional Parameter (Ascending) : ");
+var sortOrder = char.ToLower (ReadKey ().KeyChar);
+
+if (chArr.Length > 0 && chArr.All (char.IsLetter) && char.IsLetter (splCh) && (sortOrder == 'd' || sortOrder == 'o')) {
+   WriteLine ("\nThe Final Array is : ");
+   var result = sortOrder == 'o' ? GetSortArray (chArr, splCh) : GetSortArray (chArr, splCh, sort: sortOrder);
+   foreach (var ch in result) { Write (ch + " "); }
+} else WriteLine ("\nEnter Valid Input");
+
+///<summary>GetSortArray : Given a character array A, along with special character S and sort order O (default order is Ascending),
```
Comment (MetaNaveen) [inline Training/Program.cs:21]: /// is followed by a space as coding convention.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+/// print the sorted array by keeping the elements matching S to the last of the array.</summary>
+/// <param name="charr">It is the character array </param>
+/// <param name="splch">It is the spl character </param>
+/// <param name="sort">It is the sorting order</param>
+char[] GetSortArray (in char[] charr, char splch, char sort = 'a') {
+   var SortArr = charr.Where (c => c != splch);
+   SortArr = (sort == 'a') ? SortArr.Order () : SortArr.OrderDescending ();
+   return charr.Any (x => x == splch) ? SortArr.Concat (charr.Where (c => c == splch)).ToArray () : SortArr.ToArray ();
+}
+
```
Comment (MetaNaveen) [inline Training/Program.cs:31]: Unnecessary blank lines.

**Review by MetaNaveen — COMMENTED** [review]
Close the PR after fixing.

File: Training/Program.cs (hunk 2 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: Done changes closing the PR

**Pull request closed without merging**